ui/cypress/integration/slow_query/list.compat_spec.js

```diff
@@ -0,0 +1,99 @@
+// Copyright 2022 PingCAP, Inc. Licensed under Apache-2.0.
+import { skipOn } from '@cypress/skip-test'
+
+describe('SlowQuery list compatibility test', () => {
+  before(() => {
+    cy.fixture('uri.json').then(function (uri) {
+      this.uri = uri
+    })
+  })
+
+  beforeEach(function () {
+    cy.login('root')
+    cy.visit(this.uri.slow_query)
+    cy.url().should('include', this.uri.slow_query)
+  })
+
+  describe('Available fields', () => {
+    skipOn(Cypress.env('FEATURE_VERSION') !== '6.0.0', () => {
+      it('Show all available fields', () => {
+        cy.intercept('/dashboard/api/slow_query/available_fields').as(
+          'getAvailableFields'
+        )
+        cy.wait('@getAvailableFields')
+
+        const availableFields = [
+          'query',
+          'digest',
+          'instance',
+          'db',
+          'connection_id',
+          'timestamp',
+
+          'query_time',
+          'parse_time',
+          'compile_time',
+          // 'rewrite_time',
```

Comment:
How about remove the commented fields if we don't check them.

Reply:
I don't know if this will help in the review. In this way we can easily know which fields are to be displayed or not

Reply:
Maybe I should extract these as a test case.

Reply:
Got you! But actually cypress helps us to check these fields, I think we don't need to check by ourselves again. Maybe we can add the comparison of length of availableFields and the length of fields shows on popover content list.

```
cy.get(`[data-e2e^="columns_selector_field_"]`).should('have.length', availableFields.length)
```

Reply:
There might be a problem if the length of availableFields and the length of fields is the same but the fields are wrong.

Reply:
I think check both the existence of columns_selector_field_xx and length of column selector fields can be more safe.